ts/@live-compositor/browser-render/package-lock.json

```diff
@@ -1,317 +0,0 @@
-{
```

Comment:
In workspace setup there is only one lock file in the root that have all the dependencies